models/db-models.js

```diff
@@ -31,4 +31,17 @@ exports.selectCommentsByArticleId = (articleId) => {
     ORDER BY created_at DESC
   `;
   return db.query(queryString, [articleId]).then(({ rows }) => rows);
+}
+
+exports.insertCommentByArticleId = (articleId, comment) => {
+  const { body, username } = comment;
+  const timestamp = new Date(Date.now()).toISOString();
+  const queryString = `
+    INSERT INTO comments
+      (body, votes, author, article_id, created_at)
```

Comment:
I believe the table is set up to create a time_stamp for you - if you take out the created_at value and column name does it still work ?

Reply:
yep, still works without it, Thanks Mitch